docs/run/advanced-runtime-options.mdx

```diff
@@ -303,7 +303,16 @@ estimator = Estimator(session=backend, options=options)
 </TabItem>
 </Tabs>
 
-
+## Options classes
+
+|                                                 Category                                                |                             Description                            |                         Example                        |
+|:-------------------------------------------------------------------------------------------------------:|:------------------------------------------------------------------:|:------------------------------------------------------|
+| [Resilience](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.ResilienceOptionsV2)                  | Advanced options for configuring error mitigation methods such as measurement error mitigation, ZNE, and PEC. Estimator only. | `estimator.options.resilience.measure_mitigation = True` |
+| [Dynamical decoupling](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.DynamicalDecouplingOptions) | Options for dynamical decoupling.                                  | `estimator.options.dynamical_decoupling.enable = True`   |
+| [Execution](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.ExecutionOptionsV2)  |  Execution options for V2 primitives, including whether to initialize qubits and the repetition delay.   | `estimator.options.execution.init_qubits = False`   |
```

Comment:
Why is V2 primitives called out here, when most other options (like twirling) are also v2 only?

Reply:
> Why is V2 primitives called out here, when most other options (like twirling) are also v2 only?

Just because the description on the execution options page called it out. :)  I'll update it